File: GeeksForGeeks/Index Of an Extra Element/GFG.java
class GfG
{
	public int findExtra(int a[],int b[],int n)
        {
         // add code here.
         
            int i=0;
            
            try {
                while(a[i] == b[i]) {
                    i++;
                }
            }
            
            catch(ArrayIndexOutOfBoundsException e) {
                return i;
            }
            return i;
             
        
        }
}

File: GeeksForGeeks/Need Some Change/GFG.cpp
/*This is a function problem.You only need to complete the function given below*/
//User function Template for C++
/*Function to swap array elements
* arr : array input
* sizeof_array : number of elements in array
*/
void swapElements(int arr[], int sizeof_array){

// Your code here
	int temp;
	for(int ind = 0;ind < sizeof_array;ind++){
		if((ind+2) == sizeof_array) break;
		temp = arr[ind];
		arr[ind] = arr[ind+2];
		arr[ind+2] = temp;
	}

}